results/cmd/api/main.go

```diff
@@ -39,7 +41,16 @@ func main() {
 	}
 	defer db.Close()
 	srv := &server{db: db}
+	database := srv.db
 
+	statement, err := database.Prepare("CREATE TABLE IF NOT EXISTS taskrun (logid binary(16) PRIMARY KEY, taskrunlog BLOB, uid INTEGER, name TEXT, namespace TEXT)")
```

Comment:
this was moved to the .sql file right? so, we do not need this here?

Reply:
yeah, it is not shown in conflict diff so I missed it, I will also delete them in other functions. Done